src/components/views/elements/AppTile.js

```diff
@@ -556,5 +543,26 @@ export default React.createClass({
                 { appTileBody }
             </div>
         );
-    },
-});
+    }
+}
+
+AppTile.propTypes = {
+    id: React.PropTypes.string.isRequired,
+    url: React.PropTypes.string.isRequired,
+    name: React.PropTypes.string.isRequired,
+    room: React.PropTypes.object.isRequired,
+    type: React.PropTypes.string.isRequired,
+    // Specifying 'fullWidth' as true will render the app tile to fill the width of the app drawer continer.
+    // This should be set to true when there is only one widget in the app drawer, otherwise it should be false.
+    fullWidth: React.PropTypes.bool,
+    // UserId of the current user
+    userId: React.PropTypes.string.isRequired,
+    // UserId of the entity that added / modified the widget
+    creatorUserId: React.PropTypes.string,
+    waitForIframeLoad: React.PropTypes.bool,
+};
+
+AppTile.defaultProps = {
+    url: "",
+    waitForIframeLoad: true,
+};
```

Comment:
this can be written as `static propTypes = {...};` within the body of the class since we babel things, more readable than having to go to the bottom of the file to find the propTypes imho. (Same for defaultProps)